kubectl-rook-ceph.sh

```diff
@@ -123,19 +124,19 @@ function parse_flags() {
       break
       ;;
     esac
-    is_flag "$VAL" && fail_error "Flag '$FLAG' value '$VAL' looks like another flag"
+    is_flag "$VAL" && fail_error "${ERROR}Flag '$FLAG' value '$VAL' looks like another flag"
```

Comment:
Why not change `fail_error` to print the error format instead of having to add `${ERROR}` to everything?